### PR TITLE
GGA in 2023 news

### PR DESCRIPTION
**Pull request by abretaud**
Starting a draft PR for a new GGA blogpost. A lot of things to write still, but let's open it

ping @bgruening @hexylena @rlibouba at least

### CODE REVIEW

File: content/news/2023-10-26-gga-update/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| InterProScan |  |  |  |  |
+| EggNOG-Mapper |  |  |  |  |
+| Genome annotation statistics |  |  |  |  |
+| BUSCO |  |  |  |  |
+| AEGeAN |  |  |  |  |
+| BLAST |  |  |  |  |
+| Exonerate |  |  |  |  |
+| Diamond |  |  |  |  |
+| Miniprot |  |  |  |  |
+| FEELnc |  |  |  |  |
```
Comment (bgruening) [inline content/news/2023-10-26-gga-update/index.md:44]:
```suggestion
| Tool | Version | UseGalaxy.eu | UseGalaxy.org | UseGalaxy.org.au |
|---|---|---|---|---|
| RepeatModeler | 2.0.4 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/csbl/repeatmodeler/repeatmodeler/2.0.4+galaxy1)  |  |  |
| RepeatMasker | 4.1.5 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/4.1.5+galaxy0) |  |  |
| Red | 2018.09.10 | [Red](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/red/red/2018.09.10+galaxy1) |  |  |
| Prokka | 1.14.6 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/crs4/prokka/prokka/1.14.6+galaxy1)  |  |  |
| Bakta | 1.8.2 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/bakta/bakta/1.8.2+galaxy0) |  |  |
| Helixer | 0.3.2 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/genouest/helixer/helixer/0.3.2) |  |  |
| BRAKER2 | 2.1.6 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/genouest/braker/braker/2.1.6+galaxy0) |  |  |
| BRAKER3 | 3.0.3 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/3.0.3+galaxy1) |  |  |
| TSEBRA |  |  |  |  |
| Funannotate | 1.8.15 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/funannotate_predict/funannotate_predict/1.8.15+galaxy1) |  |  |
| MAKER | 2.31.11 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/maker/maker/2.31.11+galaxy2) |  |  |
| Antismash | 6.1.1 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/antismash/antismash/6.1.1+galaxy1) |  |  |
| InterProScan | 5.59-91 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/interproscan/interproscan/5.59-91.0+galaxy3) |  |  |
| EggNOG-Mapper | 2.1.8 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/eggnog_mapper/eggnog_mapper/2.1.8+galaxy4) |  |  |
| Genome annotation statistics | 0.8.4 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/jcvi_gff_stats/jcvi_gff_stats/0.8.4) |  |  |
| BUSCO | 5.4.6 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/busco/busco/5.4.6+galaxy0) |  |  |
| AEGeAN | 0.16.0 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/aegean_parseval/aegean_parseval/0.16.0) |  |  |
| BLAST | 2.14.1 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastn_wrapper/2.14.1+galaxy0) |  |  |
| Exonerate | 2.4.0 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/exonerate/exonerate/2.4.0+galaxy2) |  |  |
| Diamond | 2.0.15 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/diamond/bg_diamond/2.0.15+galaxy0) |  |  |
| Miniprot | 0.12 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/miniprot/miniprot/0.12+galaxy0) |  |  |
| FEELnc | 0.2.1  | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/feelnc/feelnc/0.2.1+galaxy0) |  |  |
| JBrowse | 1.16.11 | [link](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.11+galaxy1) |  |  |
```

File: content/news/2023-10-26-gga-update/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## New workflows!
```
Comment (bgruening) [inline content/news/2023-10-26-gga-update/index.md:46]: We just list all workflows here I think, not just the new ones.

File: content/news/2023-10-26-gga-update/index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## New visualisations!
+
+- the existing ones: jbrowse/circos
+- GNB
+  - install gxit on .eu (if not done?)
+
+## New training material!
+
+- Gallantries hall of fame / learning path etc
```
Comment (bgruening) [inline content/news/2023-10-26-gga-update/index.md:60]:
```suggestion
- Gallantries hall of fame / learning path etc

From an "Introduction to Genome Annotation" to "Refining Genome Annotations with Apollo", an leaning path for everyone that wants to learn Genome Annotation: https://training.galaxyproject.org/training-material/learning-pathways/genome-annotation-eukaryote.html

```

File: content/news/2023-10-26-gga-update/index.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+## Manual curation made easy
+
+- Apollo infra, available on usegalaxy.eu/apollo
```
Comment (bgruening) [inline content/news/2023-10-26-gga-update/index.md:64]: Take content from the older blog post?

File: content/news/2023-10-26-gga-update/index.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## EBP
+
+Building on the VGP assembly workflows, doing the same for annotation
```
Comment (bgruening) [inline content/news/2023-10-26-gga-update/index.md:68]:
```suggestion
Building on the [VGP assembly workflows](https://galaxyproject.org/projects/vgp/), doing the same for annotation
```

File: content/news/2023-10-26-gga-update/index.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+- IWC repeat masking
+- functional annotation https://github.com/galaxyproject/iwc/pull/228
+- GTN workflows: funannotate, maker, prokka
+
+## New visualisations!
+
+- the existing ones: jbrowse/circos
+- GNB
+  - install gxit on .eu (if not done?)
```
[the remark this answers is not in the document]
Comment (abretaud) [reply content/news/2023-10-26-gga-update/index.md:70]: https://toolshed.g2.bx.psu.edu/repository/view_repository?id=ef69fe5507b8d8c7&changeset_revision=bd94e502872a
and
https://gitlab.com/ifb-elixirfr/usegalaxy-fr/galaxy/-/blob/release_23.0_ifb/tools/interactive/interactivetool_genenotebook.xml

Do you have time for a quick install on .eu (I can test it) or we'll see later?

[the remark this answers is not in the document]
Comment (abretaud) [reply content/news/2023-10-26-gga-update/index.md:70]: https://github.com/usegalaxy-eu/usegalaxy-eu-tools/pull/638

**Pull request merged**